### PR TITLE
Button states

### PR DESCRIPTION
**Pull request by hailin-zhang**
#Button States for hover and outline

Status: **Ready for review!**

Trello: **https://trello.com/c/Sn4EzQmk/65-button-states-for-hover-and-outline**
Linked PRs: **(links to corresponding PRs, optional)**

## Changes
- Added toggling effects to the GUI buttons for highlighting and outlining
- Will need more investigation into the `border-color` and `font-color` attributes from https://github.com/rdub80/aframe-gui, they're not working as expected
- Note that there are edge cases with selecting/de-selecting different pathways. Not sure how best to handle these cases, since the structure is changing when different buttons are clicked

## How to test-drive this PR
- (necessary config changes)
- (necessary corresponding PRs)
- (how to access the new / changed functionality -- fixtures, URLs)
- scripts to run/instructions

## Research resources
- (link1)

## TODOS:

- [ ] Tests added
- [ ] Tests passed
- [ ] Build generated
- [ ] README.md updated

### CODE REVIEW

File: public/js/highlight-sequence.js [hunk 1]
```diff
@@ -50,30 +46,137 @@ AFRAME.registerComponent("highlight-sequence", {
...
+      if (component.clickState === 2) {
+        cleanupOutlines();
+      }
+      const clickState = (component.clickState + 1) % 3;
+      updateButtonColor(clickState, sequenceName);
+      // increment clickState attribute
+      this.el.setAttribute('highlight-sequence', 'clickState', clickState);
+      switch (clickState) {
+        case 0:
+          cleanupHighlights(sequenceName);
```
Comment (dacerron) [inline public/js/highlight-sequence.js:119]: Calling this function here makes highlights for edges in other sequences disappear too. Should we call cleanupOutlines here instead?

Comment (dacerron) [reply public/js/highlight-sequence.js:119]: image
image
image
image

Comment (dacerron) [reply public/js/highlight-sequence.js:119]: I see now that calling just cleanupOutlines won't get rid of the highlight on the bottom 2 edges. I think we may need a data structure to keep track of highlighted edges or make edges keep track of their state. I will draw up a design to present at our meeting this afternoon.

Comment (hailin-zhang) [reply public/js/highlight-sequence.js:119]: yep that makes sense, since the sequences can overlap between pathways
